install.php

```diff
@@ -1,3 +1,40 @@
 <?php
-$_GET["mod"] = "install";
-include_once("index.php");
+
+//raise error reporting - could be interesting and this is autmomatically reset when index.php is executed
+error_reporting(E_ALL);
+
+
+/**
+* Check for common failures
+*
+* Small function to check against a number of dependencies that cause install.php to fail without showing any error details
+* @return bool check result. True if error found, false if all OK
+*/
+function checkdeps(){
```

Comment:
I would suggest a more declarative name like `checkInstallationDependencies` or `checkDependencies`.

And wouldn't it make sense to place this function in an own file inside `inc/Functions/`?
With this we have a clear split. We just do a `require_once(...)` here.

Reply:
I was in doubt if I should even write a function. 
Not like the code is repeatly called or could be used elsewhere.
But thinking about it now :
This issues could re-appear during upgrade scenarios (e.g. unpack new release as root and forget to chown afterwards).
In that case it would fit more into index.php. 
But on the other hand I don't really want to run that check on every execution.
So we could also put it into the install class as static function (to avoid creating an object with its dependencies) and provide an full environment checker (that also runs the other install checks) instead.
Your opinion?